docs/decisions/single-glossary.md

```diff
@@ -0,0 +1,11 @@
+# Glossary Scope
+
+**Decision Date:** February 2021
+
+**Summary:** A single glossary would be used for terms from all systems.
+
+## Details
+
+We initially started our effort with a glossary defined per system (including a top level system glossary).
+
+Through time we aligned on wanting to use the same language across systems and components,to avoid different terms meaning different things in different contexts.
```

Comment:
:nerd_face: 

```suggestion
Through time we aligned on wanting to use the same language across systems and components, to avoid different terms meaning different things in different contexts.
```